examples/reference/streams/bokeh/Tap.ipynb

```diff
@@ -7,7 +7,7 @@
     "<div class=\"contentcontainer med left\" style=\"margin-left: -50px;\">\n",
     "    <dl class=\"dl-horizontal\">\n",
     "      <dt>Title</dt> <dd> HeatMap Tap stream example</dd>\n",
-    "      <dt>Description</dt> <dd>A linked streams example demonstrating how use Tap stream on a HeatMap. The data contains the incidence of measles across US states by year and week (obtained from [Project Tycho](http://www.tycho.pitt.edu/)). The HeatMap represents the mean measles incidence per year. On tap the Histogram on the right will generate a Histogram of the incidences for each week in the selected year and state.</dd>\n",
+    "      <dt>Description</dt> <dd>A linked streams example demonstrating how use Tap stream on a HeatMap. The data contains the incidence of measles across US states by year and week (obtained from <a href="http://www.tycho.pitt.edu">Project Tycho</a>). The HeatMap represents the mean measles incidence per year. On tap the Histogram on the right will generate a Histogram of the incidences for each week in the selected year and state.</dd>\n",
```

Comment:
You broke this notebook here, took a long time to figure out. I recommend against editing notebook JSON manually.

Reply:
D'oh!  Yes, it's more trouble than it saves.  Needed \ " here...